<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphDialog</class>
 <widget class="QDialog" name="GraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph of Transition Matrix</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="GraphWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>651</width>
     <height>651</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelScaleMin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelScaleMaxY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelScaleMaxX">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>26</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale Max</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderRGBVar">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelOne">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTwo">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelThree">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Colour Ranges</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adjust:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>graphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
